.github/workflows/check-cherry-pick.yml

```diff
@@ -0,0 +1,76 @@
+name: "Check cherry-picks"
+on:
+  pull_request_target:
```

Comment:
Shouldn't this be set to only run on the release branches?

Reply:
_Possibly_, though then I considered whether we'd *ever* want someone referencing something that isn't in master or staging, and then thought there may be cases when someone cherry-picks something from staging to master if we decide to fast-track it.

Arguable.

Reply:
Does that happen often enough to justify running the action (with a full depth checkout that takes several minutes) on every push against non-release branches? Likely not.